src/main/java/com/chikli/hudson/plugin/naginator/NaginatorPublisherScheduleAction.java

```diff
@@ -242,7 +244,7 @@ public Boolean call() throws Exception {
         return false;
     }
     
-    private class InterruptibleCharSequence implements CharSequence {
+    private static class InterruptibleCharSequence implements CharSequence {

```

Comment:
<BugInstance type='SIC_INNER_SHOULD_BE_STATIC' priority='Normal' category='PERFORMANCE' message='Should com.chikli.hudson.plugin.naginator.NaginatorPublisherScheduleAction$InterruptibleCharSequence be a _static_ inner class?' lineNumber='248'/>